### ISSUE TITLE
aws-android-sdk-logs error

### ISSUE THREAD

**Issue by eashevchenko** [aws/aws-sdk-android#453]
When i ve tried to send logs from Android client to Cloudwatch i ve received next error: 

com.amazonaws.AmazonServiceException: Credential should be scoped to correct service: 'logs'.  (Service: AmazonCloudWatchLogs; Status Code: 400; Error Code: InvalidSignatureException;

i m using aws-android-sdk-logs library. 
Client has access and secret key, i ve setted correct region...
What i m doing wrong ?

**Comment by kvasukib** [comment]
Thank you @eashevchenko for reporting to us. What version of SDK are you using? Can you share a code snippet and the stacktrace?

**Comment by eashevchenko** [comment]
I m using aws-android-sdk-logs:2.6.18 . And i have implemented client like this: https://stackoverflow.com/questions/36876563/amazon-cloud-watch-log-putlogeventsrequest-the-given-sequencetoken-is-invali

**Comment by rohandubal** [comment]
Hello @eashevchenko 

This is a bud due to incorrect configuration for signer. It will be fixed in next release of the SDK.

Thanks,
Rohan

**Comment by olehspavl** [comment]
Before Rohan fix that you can use a temporal solution: set ServiceName directly.
The issue is that AmazonCloudWatchLogsClient use wrong service name.
By def service name will be initialized using def httpClientName. To avoid it you can set ServiceName by yourself:
```
AmazonCloudWatchLogsClient client =  new AmazonCloudWatchLogsClient(credential);
client.setServiceNameIntern("logs");
// .. use client after ..
```

**Comment by minbi** [comment]
Fixed in version 2.6.19
